src/_forms.css

```diff
@@ -14,3 +14,91 @@
   border: 0;
   padding: 0;
 }
+
+/* Checkboxes */
```

Comment:
we can't put stuff like this in tachyons - it goes against the purpose of the framework, which is to give us reusable, composable discrete classes that are element and usage agnostic. this is semantic css. you're also hardcoding values, not using tachyons variables.

the tachyons way would be to combine classes into a checkbox "component", like the example below:

http://tachyons.io/components/forms/newsletter-subscription/index.html